projects/1.18/assets/epic-samurais/epicsamurai/lang/zh_cn.json

```
@@ -0,0 +1,127 @@
+    "item.epicsamurai.diamond_samurai_helmet": "钻石武士头盔",
+    "item.epicsamurai.diamond_samurai_chestplate": "钻石武士胸甲",
+    "item.epicsamurai.diamond_samurai_leggings": "钻石武士护腿",
+    "item.epicsamurai.diamond_samurai_boots": "钻石武士靴",
+    "item.epicsamurai.netherite_samurai_helmet": "下界合金武士头盔",
+    "item.epicsamurai.netherite_samurai_chestplate": "下界合金胸甲",
+    "item.epicsamurai.netherite_samurai_leggings": "下界合金护腿",
+    "item.epicsamurai.netherite_samurai_boots": "下界合金武士靴",
+    "item.epicsamurai.red_samurai_helmet": "红宝石武士头盔",
+    "item.epicsamurai.red_samurai_chestplate": "红宝石武士胸甲",
+    "item.epicsamurai.red_samurai_leggings": "红宝石武士护腿",
+    "item.epicsamurai.red_samurai_boots": "红宝石武士靴",
+    "item.epicsamurai.green_samurai_helmet": "玉制武士头盔",
+    "item.epicsamurai.green_samurai_chestplate": "玉制武士胸甲",
+    "item.epicsamurai.green_samurai_leggings": "玉制武士护腿",
+    "item.epicsamurai.green_samurai_boots": "玉制武士靴",
+    "item.epicsamurai.blue_samurai_helmet": "海蓝宝石武士头盔",
+    "item.epicsamurai.blue_samurai_chestplate": "海蓝宝石武士胸甲",
+    "item.epicsamurai.blue_samurai_leggings": "海蓝宝石武士护腿",
+    "item.epicsamurai.blue_samurai_boots": "海蓝宝石武士靴",
+    "item.epicsamurai.white_samurai_helmet": "银武士头盔",
+    "item.epicsamurai.white_samurai_chestplate": "银武士胸甲",
+    "item.epicsamurai.white_samurai_leggings": "银武士护腿",
+    "item.epicsamurai.white_samurai_boots": "银武士靴",
+    "item.epicsamurai.gray_samurai_helmet": "玛瑙武士头盔",
+    "item.epicsamurai.gray_samurai_chestplate": "玛瑙武士胸甲",
+    "item.epicsamurai.gray_samurai_leggings": "玛瑙武士护腿",
+    "item.epicsamurai.gray_samurai_boots": "玛瑙武士靴",
+    "item.epicsamurai.straw_hat": "草帽",
+    "item.epicsamurai.akaname_spawn_egg": "垢尝  [WIP]",
+    "item.epicsamurai.enenra_spawn_egg": "烟罗罗  [WIP]",
+    "item.epicsamurai.oni_spawn_egg": "生成鬼怪物蛋 [WIP]",
+    "item.epicsamurai.kitsune_spawn_egg": "狐妖刷怪蛋",
+    "item.epicsamurai.onibi_spawn_egg": "鬼火刷怪蛋",
+    "item.epicsamurai.ruby": "红宝石",
+    "item.epicsamurai.jade": "玉",
+    "item.epicsamurai.onyx": "玛瑙",
+    "item.epicsamurai.aquamarine": "海蓝宝石",
+    "item.epicsamurai.silver_ingot": "银锭",
+    "item.epicsamurai.raw_silver": "粗银",
+    "item.epicsamurai.silver_nugget": "银粒",
+    "item.epicsamurai.steel_ingot": "钢锭",
+    "item.epicsamurai.steel_nugget": "钢粒",
+    "item.epicsamurai.amethyst_ingot": "紫水晶涂层钢锭",
+    "item.epicsamurai.quartz_ingot": "石英涂层钢锭",
+    "item.epicsamurai.katana": "武士刀",
+    "item.epicsamurai.naginata": "薙刀",
+    "item.epicsamurai.nagamaki": "长卷",
+    "item.epicsamurai.kunai": "苦无",
+    "item.epicsamurai.kama": "忍者镰刀",
+    "item.epicsamurai.sai": "十手",
+    "item.epicsamurai.shuriken": "飞镖",
+    "item.epicsamurai.knive_handle": "匕首手柄",
+    "item.epicsamurai.spear_handle": "长矛手柄",
+    "item.epicsamurai.katana_handle": "武士刀手柄",
+    "item.epicsamurai.steel_helmet": "钢武士头盔",
+    "item.epicsamurai.steel_chestplate": "钢武士胸甲",
+    "item.epicsamurai.steel_leggings": "钢武士护腿",
+    "item.epicsamurai.steel_boots": "钢武士靴",

```

```suggestion
    "item.epicsamurai.steel_helmet": "钢头盔",
    "item.epicsamurai.steel_chestplate": "钢胸甲",
    "item.epicsamurai.steel_leggings": "钢护腿",
    "item.epicsamurai.steel_boots": "钢靴子",
```
仅根据原文判断，还是要在游戏内实测
同时推荐所有的`武士靴`都改为`武士靴子`

---

注意修改